### PR TITLE
improve peazip translation

### PR DESCRIPTION
**Pull request by wbsdty331**
fixed transcode.

### CODE REVIEW

File: PeaZip/lang/chs.txt [hunk 1]
```diff
@@ -1,28 +1,28 @@
-﻿=== PeaZip language file ===
+=== PeaZip language file ===
 Chinese - 简体中文
 6.1
-翻译: Aerowolf, Junde Yhi
-Last rev. by: Junde Yhi
+翻译: Aerowolf, Junde Yi
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Add yourself there. @wbsdty331

File: PeaZip/lang/chs.txt [hunk 2]
```diff
@@ -265,15 +265,15 @@ txt_3_1_systmp: 系统临时文件夹
 txt_3_1_languagetools: 翻译
 txt_3_1_workingdir: 工作目录
 txt_3_0_nonreadableorpw: 压缩无法读取。输入密码试试？
-txt_3_0_readablepw: 压缩文档似乎有密码保护
+txt_3_0_readablepw: 压缩文件似乎有密码保护
 txt_3_0_configure: 系统集成
 txt_3_0_resettmp: 重置 PeaZip 临时文件夹
 txt_2_9_address: 地址栏
-txt_2_9_adv: 管理任何 7z 或 FreeArc 后台支持的文件，都可以使用高级过滤，请参见文档，并且不接受基本过滤（用于搜索功能，并显示在书签和历史记录面板中）
+txt_2_9_adv: 管理任何 7z 或 FreeArc 后台支持的文件，都可以使用高级过滤，请参见文件，并且不接受基本过滤（用于搜索功能，并显示在书签和历史记录面板中）
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Pretty sure it was "documentation" in the original text.

File: PeaZip/lang/chs.txt [hunk 3]
```diff
@@ -282,71 +282,71 @@ txt_2_9_movehere: 移动到此处
...
+txt_2_9_organize: 视图(&O)
 txt_2_9_public: 公用
 txt_2_9_rec: 递归扫描：在子目录中搜索，需要点时间
 txt_2_9_selected: 选定
-txt_2_9_setapps: 组织应用程序
+txt_2_9_setapps: 视图应用程序
 txt_2_9_showmenu: 显示菜单栏
 txt_2_9_st: 较小
-txt_2_9_test_pw2G: 测试小于 2GB 的加密的压缩文档
+txt_2_9_test_pw2G: 测试小于 2GB 的加密的压缩文件
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Double and unnecessary "的".

File: PeaZip/lang/chs.txt [hunk 4]
```diff
@@ -282,71 +282,71 @@ txt_2_9_movehere: 移动到此处
...
-txt_2_7_pwextset: 已设密码，用于解压、查看、测试已加密的压缩文档
-txt_2_7_archivehint: 右击可以添加文件到压缩文档、查看其它功能，或者也可以拖拽到此处进行添加
+txt_2_7_pwextset: 已设密码，用于解压、查看、测试已加密的压缩文件
+txt_2_7_archivehint: 右击可以添加文件到压缩文件、查看其它功能，或者也可以拖拽到此处进行添加
 txt_2_7_exthint: 右击可以添加文件来解压、查看其它功能，或者也可以将文件拖放到这里进行解压
 txt_2_7_setadvf: 设置高级过滤器
 txt_2_7_selpath: 选中的项目的路径
-txt_2_7_separateerror: 抱歉，使用“添加每个对象到单独的压缩文档”开关时，不能导入作业定义的命令行，这是因为作业是被作为多个单独的命令执行的
-txt_2_7_noinput: 视图为空：请使用添加文件或载入视图来显示要解压的压缩文档的列表
+txt_2_7_separateerror: 抱歉，使用“添加每个对象到单独的压缩文件”开关时，不能导入作业定义的命令行，这是因为作业是被作为多个单独的命令执行的
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Better translation possible.

File: PeaZip/lang/chs.txt [hunk 5]
```diff
@@ -1122,50 +1122,50 @@ F10 	菜单
...
 - "I/O"标签用于设置输入/输出选项；
-- "过滤器"标签用于设置使用高级排除过滤器，如果可以选择压缩文档格式；
+- "过滤器"标签用于设置使用高级排除过滤器，如果可以选择压缩文件格式；
 - "控制台"标签用于发现并自定义作业语法，或输出作业为命令行。
 
-或者，在单个过程中，您可以从系统右键菜单中添加每个输入项目到单独的压缩文档中，并且从发送到菜单您可以添加多个项目到一个压缩文档中。
+或者，在单个过程中，您可以从系统右键菜单中添加每个输入项目到单独的压缩文件中，并且从发送到菜单您可以添加多个项目到一个压缩文件中。
 
-中压缩文档视图中您也可以:
+中压缩文件视图中您也可以:
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Does not sound right.

File: PeaZip/lang/chs.txt [hunk 6]
```diff
@@ -737,18 +737,18 @@ txt_on: 位于
...
+txt_openarchive: 打开压缩文件
+txt_title_open: 打开压缩文件、解密、合并...
 txt_open_bookmark: 打开书签
 txt_cphere: 在此打开命令提示符
 txt_open_file: 打开文件
 txt_open_files: 打开文件
 txt_open_path: 打开路径
-txt_opensource: 打开便携压缩文档
+txt_opensource: 打开便携压缩文件
 txt_openwith: 打开方式...
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Use … for god sakes...

File: PeaZip/lang/chs.txt [hunk 7]
```diff
@@ -1045,56 +1045,56 @@ txt_time: 时间：
...
-- 右击压缩文档，在右键菜单中选择“解压到此处”或“解压到此处（到新文件夹中）”来直接解压缩
+- 右击压缩文件，在右键菜单中选择“解压到此处”或“解压到此处（到新文件夹中）”来直接解压缩
 - 另外，请使用“解压...”菜单可以有更多选项：输出路径，密码，解压到新文件夹，选择路过、重命名或覆盖已有的文件等。
 
-要在PeaZip中打开压缩文档，请双击，或者拖拽压缩文档的到PeaZip窗口或图标上
+要在PeaZip中打开压缩文件，请双击，或者拖拽压缩文件的到PeaZip窗口或图标上
 - 在工具栏或右键菜单中点击“解压”可以看到包含所有选项的确认对话框（输出路径、密码、命名规则、解压到新文件夹等），但仅可以解压选定的对象
 - 拖拽文件或文件夹到某处，可以只解压选定的单个内容（更多信息详见PeaZip中的拖拽章节）
-- 点击工具栏中“解压”右侧的下拉按钮，可以直接解压整个压缩文档到常用的几个位置（不必确认可以跳过确认对话框）并设置更多重要选项；使用快捷键：Ctrl+E（或F12）可以解压整个压缩文档，需要指定输出路径；Ctrl+Alt+E解压文档到当前文件夹；Ctrl+Shift+E解压到桌面；Ctrl+Alt+Shift+E解压到文档中；Ctrl+1..8解压到书签中的1..8路径（如果已经定义）
+- 点击工具栏中“解压”右侧的下拉按钮，可以直接解压整个压缩文件到常用的几个位置（不必确认可以跳过确认对话框）并设置更多重要选项；使用快捷键：Ctrl+E（或F12）可以解压整个压缩文件，需要指定输出路径；Ctrl+Alt+E解压文件到当前文件夹；Ctrl+Shift+E解压到桌面；Ctrl+Alt+Shift+E解压到文件中；Ctrl+1..8解压到书签中的1..8路径（如果已经定义）
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Space around English text please... And wrong parentheses used.

File: PeaZip/lang/chs.txt [hunk 8]
```diff
@@ -1045,56 +1045,56 @@ txt_time: 时间：
...
 
-打开PeaZip，选择一或多个压缩文档，点击工具栏或右键菜单中的“解压”，或者使用快速解压可以解压到上次使用的目标位置。
+打开PeaZip，选择一或多个压缩文件，点击工具栏或右键菜单中的“解压”，或者使用快速解压可以解压到上次使用的目标位置。
 
-仅解压文档中的选定文件
-要在PeaZip中打开压缩文档，请双击，或者拖拽压缩文档的到PeaZip窗口或图标上
+仅解压文件中的选定文件
+要在PeaZip中打开压缩文件，请双击，或者拖拽压缩文件的到PeaZip窗口或图标上
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Same here.

File: PeaZip/lang/chs.txt [hunk 9]
```diff
@@ -1045,56 +1045,56 @@ txt_time: 时间：
 
 解压缩
 在系统中
-- 右击压缩文档，在右键菜单中选择“解压到此处”或“解压到此处（到新文件夹中）”来直接解压缩
+- 右击压缩文件，在右键菜单中选择“解压到此处”或“解压到此处（到新文件夹中）”来直接解压缩
 - 另外，请使用“解压...”菜单可以有更多选项：输出路径，密码，解压到新文件夹，选择路过、重命名或覆盖已有的文件等。
 
-要在PeaZip中打开压缩文档，请双击，或者拖拽压缩文档的到PeaZip窗口或图标上
+要在PeaZip中打开压缩文件，请双击，或者拖拽压缩文件的到PeaZip窗口或图标上
```
Comment (MingcongBai) [inline PeaZip/lang/chs.txt]: Spaces.

**Comment by MingcongBai** [comment]
Could you rebase your PR on the latest changes? My apologies.

**Comment by lmy441900** [comment]
@MingcongBai rebase 也会炸掉

**Comment by MingcongBai** [comment]
Should I reset here or...?

**Comment by lmy441900** [comment]
@wbsdty331 Thanks for your contribution. Here are some problems we've found in your PR:

1. `.reg` file must be encoded as **GBK** or **GB18030**. This PR changes it into **UTF-8**.
2. `chs.txt` must be encoded as **UTF-8 w/ BOM**. This PR changes it into **UTF-8 w/o BOM**.
3. Base is not the newest; some new fixes is made by @MingcongBai . You may first merge them into your changes.

Problem 1 and 2 can cause these localization files completely unusable. We suggest you to have a test first.

The following fixes in your PR are nice:

- Unifying the translation of "archive". This is what I didn't do.
- Changing `[]` into `()`.
- Slight fixes of some words.
- Maybe more.

Since your changes are privided as 3 "big" commits, we're afraid that we cannot merge it at this moment. You can fix the problems talked above first.

**Comment by MingcongBai** [comment]
Or we can take the suggestion and thank him for it?

**Comment by wbsdty331** [comment]
I will have them as soon as possible.

**Comment by MingcongBai** [comment]
Thanks.

**Pull request closed without merging**